### PR TITLE
Add unbranded negative spinner

### PR DESCRIPTION
**Pull request by calvin-fung-zopa**
Add unbranded theme for spinner negative:

Screenshot 2022-06-16 at 10 12 28

### CODE REVIEW

File: src/components/atoms/Spinner/Spinner.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import actionSpinner from './spinner-action.gif';
 import deprecated from 'deprecated-prop-type';
 import { useThemeContext } from '../../styles/Theme';
 import jlSpinner from '../../../content/images/jl-spinner.gif';
```
Comment (admmasters) [inline src/components/atoms/Spinner/Spinner.tsx:7]: I guess we shouldn't reference `jl` specifically in the open source lib, maybe we should rename this to _something_

Comment (calvin-fung-zopa) [reply src/components/atoms/Spinner/Spinner.tsx:7]: shall we call it 'nonBrandedSpinner' ?

Comment (calvin-fung-zopa) [reply src/components/atoms/Spinner/Spinner.tsx:7]: since its just black and white it would make sense to refer to it as nonBranded

Comment (calvin-fung-zopa) [reply src/components/atoms/Spinner/Spinner.tsx:7]: Bear in mind we implemented it like this though ` const animationData = theme?.spinner?.spinnerTheme === 'jl'`. Should we change this too?

Comment (calvin-fung-zopa) [reply src/components/atoms/Spinner/Spinner.tsx:7]: We could do `theme?.spinner?.spinnerTheme === 'unBranded`?

**Review by admmasters — APPROVED** [review]
LGTM!

**Pull request merged**